src/boolean_result.cpp

```diff
@@ -879,8 +879,8 @@ Manifold::Impl Boolean3::Result(OpType op) const {
 
   // Level 6
 
-  if (ManifoldParams().intermediateChecks)
-    DEBUG_ASSERT(outR.IsManifold(), logicErr, "polygon mesh is not manifold!");
+  // if (ManifoldParams().intermediateChecks)
```

Comment:
The problem with this is that we are checking for polygon mesh manifoldness, and the tightened manifoldness check is not suitable for that (we assume a triangular mesh).

Reply:
I think it's fine to just remove this instead of commenting it out. I can't remember the last time it caught anything anyway.